OpenOversight/tests/test_utils.py

```diff
@@ -337,3 +338,21 @@ def test_filter_by_form_filter_unit(
         if has_officers_with_no_unit:
             found = found or any([a.unit_id is None for a in officer.assignments])
         assert found
+
+
+@pytest.mark.parametrize(
+    "url,is_valid",
+    [
+        ("/images/1", True),
```

Comment:
This is a little pedantic but do you mind adding one more case for if query params exist?
```suggestion
        ("/images/1", True),
        ("/images/1?with_param=true", True),
```

Reply:
Done!